src/common/legacy_config_opts.h

```diff
@@ -188,128 +178,7 @@ OPTION(ms_dpdk_debug_allow_loopback, OPT_BOOL)
 OPTION(ms_dpdk_rx_buffer_count_per_core, OPT_INT)
 
 OPTION(inject_early_sigterm, OPT_BOOL)
-
-OPTION(mon_data, OPT_STR)
-OPTION(mon_initial_members, OPT_STR)    // list of initial cluster mon ids; if specified, need majority to form initial quorum and create new cluster
-OPTION(mon_compact_on_start, OPT_BOOL)  // compact leveldb on ceph-mon start
-OPTION(mon_compact_on_bootstrap, OPT_BOOL)  // trigger leveldb compaction on bootstrap
-OPTION(mon_compact_on_trim, OPT_BOOL)       // compact (a prefix) when we trim old states
-OPTION(mon_osd_cache_size, OPT_INT)  // the size of osdmaps cache, not to rely on underlying store's cache
-
-OPTION(mon_cpu_threads, OPT_INT)
-OPTION(mon_osd_mapping_pgs_per_chunk, OPT_INT)
-OPTION(mon_osd_max_creating_pgs, OPT_INT)
-OPTION(mon_tick_interval, OPT_INT)
-OPTION(mon_session_timeout, OPT_INT)    // must send keepalive or subscribe
-OPTION(mon_subscribe_interval, OPT_DOUBLE)  // for legacy clients only
-OPTION(mon_delta_reset_interval, OPT_DOUBLE)   // seconds of inactivity before we reset the pg delta to 0
-OPTION(mon_osd_laggy_halflife, OPT_INT)        // (seconds) how quickly our laggy estimations decay
-OPTION(mon_osd_laggy_weight, OPT_DOUBLE)          // weight for new 'samples's in laggy estimations
-OPTION(mon_osd_laggy_max_interval, OPT_INT)      // maximum value of laggy_interval in laggy estimations
-OPTION(mon_osd_adjust_heartbeat_grace, OPT_BOOL)    // true if we should scale based on laggy estimations
-OPTION(mon_osd_adjust_down_out_interval, OPT_BOOL)  // true if we should scale based on laggy estimations
-OPTION(mon_osd_auto_mark_in, OPT_BOOL)         // mark any booting osds 'in'
```

Comment:
@shinobu-x not all descriptions migrated.

Reply:
@shinobu-x this comment is not addressed. please ping me again, once you've finished doing so.